src/server.js

```diff
@@ -151,14 +158,14 @@ const handleNavTreeEndpoint = (req, res) => {
   });
 };
 
-const handleCollectionRender = (req, res) => {
+const handleCollectionRender = async (req, res) => {
   const { id } = req.params;
   if (!ObjectId.isValid(id)) {
     throw new Error(`id '${id}' is not valid`);
   }
-  getCollection(id, (collection) =>
-    handleRender(req, res, { current: collection }),
-  );
+  const queryClient = new QueryClient();
```

Comment:
Is it necessary to create a new `QueryClient` object each time you want to access it? Would it be possible to define it globally?

Reply:
Yes for security reasons. I quote 

> Create a new QueryClient instance inside of your request handler. This ensures that data is not shared between different users and requests.